pkgs/development/python-modules/pymssql/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, freetds, cython, setuptools-git }:
+
+buildPythonPackage rec {
+  pname = "pymssql";
+  version = "2.1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1yvs3azd8dkf40lybr9wvswvf4hbxn5ys9ypansmbbb328dyn09j";
+  };
+
+  buildInputs = [cython setuptools-git];
+  propagatedBuildInputs = [freetds];
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
The tests require an SQL Server instance, so I just skipped them.
If that's OK, I'll add the comment right away

Reply:
@FRidh I added the comment. Let me know if that's all